app/services/dialog_local_service.rb

```diff
@@ -55,8 +19,6 @@ def determine_dialog_locals_for_svc_catalog_provision(resource_action, target, f
   def determine_dialog_locals_for_custom_button(obj, button_name, resource_action, display_options = {})
     dialog_locals = {:force_old_dialog_use => true}
```

Comment:
I don't think we need this anymore since we're not short-circuiting on the next line.